ARExperiment/README.md

```diff
@@ -0,0 +1,32 @@
+# ARExperiment
+
+This is a simple demo app that pairs with our investigation into ARKIT
+
+* `OneModelUsingAnchorsViewController` Adds one 3D model to the scene using `ARAnchors` method described on the [Getting started with ARKit](https://www.novoda.com/blog/getting-started-with-arkit/) blog post
+* `OneModelUsingVectorsViewController` Adds one 3D model to the scene using `SCNNodes` and `planeDetection` method described on the [Getting started with ARKit](https://www.novoda.com/blog/getting-started-with-arkit/) blog post
+* `SizeComparisonViewController` Adds 3 different sized cubes to the `sceneView` to check how big cm are in the AR world
+* `RecognizeObjectsViewController` Adds a 3D model to the scene using CoreML to recognise where to place the model. You can read more on the Machine Learning part in the [Making AR more precise with CoreML](https://www.novoda.com/blog/arkit-coreml/)
+* `LightsAnimationsViewController` Adds one 3D model to the scene using `ARAnchors` and allowing for animations, lights and shadows to be imported from the model
+
+## FileNames vs NodeNames
+
+At the beginning of every controller you will see there are around 3 or more string variables hardcoded
+  - `fileName` if you are assets are not on the route of `art.scnassets` folder you should add the folder name before the file name like so: `Banana/banana`
```

Comment:
your assets are not located in `arc.scnassets`?

Reply:
no, they go inside `art.scnassets` according to Apple documentation

Reply:
i meant as a suggestion, sorry. so `(...) if your assets are not located in arc.scnassets folder (...)`

Reply:
the issue is that `art.scnassets` is hardcoded as a string on the files. I will add a disclaimer

Reply:
and never apologize @takecare I really appreciate your comments!